modules/scylladb/scylladb.go

```diff
@@ -0,0 +1,142 @@
+package scylladb
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+
+	"github.com/docker/go-connections/nat"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	port           = "9042/tcp"
+	shardAwarePort = "19042/tcp"
```

Comment:
question: should we expose these ports so that client code can generate URLs based on them? I see an example test using one of them. If not, we probably want a container method returning the connection string for the given port.